README.md

```diff
@@ -468,6 +469,158 @@ ANSWERS.addComponent('DirectAnswer', {
 })
 ```
 
+## Custom Direct Answer Card
+
+If you need the full power of an SDK component, you can specificy a custom Direct Answers card.
+This component will be given the directAnswer data from the query, the same data the built-in component
+operates on. You can expect that object to look something like the below:
+
+```js
+{
+  type: "FIELD_VALUE",
+  answer: {
+    entityName: "Entity Name",
+    fieldName: "Phone Number",
+    fieldApiName: "mainPhone",
+    value: "+11234567890",
+    fieldType: "phone" 
+  },
+  relatedItem: { 
+    verticalConfigId: 'people',
+    data: { ... }
```

Comment:
Flesh this out a bit. The data contains all the other fieldValues, right? People will want to know how to access those.